### DISCUSSION TITLE
Add more gun modifications.

### DISCUSSION THREAD

**Discussion by RanarTheWrathful** [Taureon/aps-plus-plus#214] [Suggestions]
Let me know if some of these do exist(and how to do them), I may have overlooked it.
- colored guns
- changing whether gun shows below or above tank, similar to turrets.
- Able to make the gun into a shape if you don't want to use aspect, width and height to build it.
Thank you.

**Comment by Frostbyte178** [comment]
only one missing is the last one

**Comment by RanarTheWrathful** [comment]
How about the ALPHA of the gun?

**Comment by Trioplane** [comment]
not yet but will be probably easy to add
imma try
